packages/ember-htmlbars/lib/helper.js

```diff
@@ -41,6 +41,33 @@ import Object from 'ember-runtime/system/object';
 
   Additionally, class helpers can call `recompute` to force a new computation.
 
+  If you only require a simple functional style helper (where the output is only
+  dependent on the current input) then you can use the `Helper.helper` function.
+
+  In this form the example above becomes:
+  
+  ```js
+  export default Ember.Helper.helper((params, hash) => {
+    let cents = params[0];
+    let currency = hash.currency;
+    return `${currency}${cents * 0.01}`;
+  });
+  ```
+
+  This latter form is very slightly more efficient at run time and uses fewer characters
```

Comment:
@BryanCrotaz I'm curious if you can revisit this sentance. I get the point that it can be more efficient at run time. I'm wondering if the phrase `is very slightly more efficient` can be shorter. Maybe just a brief statement like "This latter form can be more efficient at run time, and exporting the function makes it easy to test."